app/views/shared/_host_next_toc.html.erb

```diff
@@ -0,0 +1,49 @@
+<section>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-md-3"></div>
+      <div class="col-md-6" class="center-block">
+        <h1 class="text-center">Host next TOC</h1>
+        <h3 class="text-center">Talk to us</h3>
+        <form>
+          <div class="form-group">
+            <input type="first_name" class="form-control" placeholder="First Name">
+          </div>
+          <div class="form-group">
+            <input type="last_name" class="form-control" placeholder="Last Name">
+          </div>
+          <div class="form-group">
+            <input type="company_name" class="form-control" placeholder="Company Name">
+          </div>
+          <div class="form-group">
+            <input type="email" class="form-control" placeholder="Email">
+          </div>
+          <div class="form-group">
+            <input type="date" id="myDate" class="form-control" placeholder="Date">
+          </div>
+          <div class="form-group">
+            <input list="participants" class="form-control" placeholder="Number of participants">
+              <datalist id="participants">
+                <option value="5">
```

Comment:
use the same values as in the cost calculation, and also, please write valid HTML: 

`<option value='20'>20</option>`